sdk/python/feast/client.py

```diff
@@ -348,7 +367,7 @@ def field_values_to_df(
                         value_attr[feature_id]
                     ]
```

Comment:
If `FEAST_VALUE_ATTR_TO_DTYPE` had a mapping of `None` to `None`, this if statement is not necessary.

Reply:
Yip. The whole type_map.py file needs to be cleaned up actually. A lot of debt in there. Still a pending task.

Reply:
I removed this whole block of code. It was dead anyways. Problem solved!